h2o-py/h2o/__init__.py

```diff
@@ -25,13 +25,10 @@
 
 __version__ = "SUBST_PROJECT_VERSION"
 
-# __all__ = ['assembly', 'astfun', 'connection', 'cross_validation', 'display',
-#            'expr', 'frame', 'group_by', 'h2o',
-#            'job', 'two_dim_table', 'estimators', 'grid', 'model', 'transforms']
 
 __all__ = ("connect", "init", "api", "connection", "upload_file", "lazy_import", "import_file", "import_sql_table",
-           "import_sql_select", "parse_setup", "parse_raw", "assign","deep_copy" "get_model", "get_grid", "get_frame",
+           "import_sql_select", "parse_setup", "parse_raw", "assign", "deep_copy", "get_model", "get_grid", "get_frame",
```

Comment:
One can set up automatic linting in Intellij or Sublime, but we don't have any lint checks on commit (that's a Phabricator's feature).